controllers/v1/userCoursesController.js

```
@@ -0,0 +1,147 @@
+        //passing true for update userCourses doucment with isDeleted true
+        let userCoursesData = await userCoursesHelper.update(req.body, req.userDetails, true);
+        userCoursesData['result'] = userCoursesData.data;
+        return resolve(userCoursesData);
+      } catch (error) {
+        return reject({
+          status: error.status || httpStatusCode.internal_server_error.status,
+          message: error.message || httpStatusCode.internal_server_error.message,
+          errorObject: error,
+        });
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)